include/rdma/fabric.h

```diff
@@ -333,6 +333,13 @@ enum {
 	FI_PROTO_SM2,
 };
 
+enum {
+	FI_TAG_BITS,
+	FI_TAG_HPC,
+	FI_TAG_AI,
```

Comment:
The above should revert back to FI_TAG_MPI and FI_TAG_NCCL, respectively, as the tag formats specifically target the tag definitions for that middleware.

Reply:
FI_TAG_CCL to be more implementation neutral.